api/config_entry_gateways.go

```diff
@@ -37,6 +40,12 @@ type IngressGatewayConfigEntry struct {
 	ModifyIndex uint64
 }
 
+type IngressServiceConfig struct {
+	MaxConnections        *int
+	MaxPendingRequests    *int
+	MaxConcurrentRequests *int
```

Comment:
Any reason these are `int` instead of `uint32`?

Reply:
Fixed. Thanks.